### PR TITLE
Plm 536 datadog deployment issue

### PR DESCRIPTION
**Pull request by evgenygigi**

### CODE REVIEW

**Review by Eli-Meitner — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,100 +1,114 @@
...
-https://docs.datadoghq.com/logs/guide/forwarder/
+# AWS Datadog integration module.
 
+Terraform module which creates Datadog integration on AWS.
+
+## <ins>Usage</ins>
+```hcl
+module "datadog" {
+  source                      = "toluna-terraform/datadog-integration/aws"
+  loggroup_envs               = [<list of strings>]
```
Comment (raanan245) [inline README.md]: That something we plan to fix on the next release....   right ? we have decided to have each service define its own log-group name, and not guess it in this module. Please open a ticket for this and log it here.

File: main.tf [hunk 2]
```diff
@@ -74,37 +75,29 @@ resource "datadog_integration_aws" "integration" {
...
   metrics_collection_enabled       = "true"
   resource_collection_enabled      = "true"
-  account_specific_namespace_rules = {
-  }
+  account_specific_namespace_rules = {}
 }
 
 # Create a new Datadog - Amazon Web Services integration Lambda ARN.
 # https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_aws_lambda_arn
 resource "datadog_integration_aws_lambda_arn" "main_collector" {
```
Comment (raanan245) [inline main.tf:86]: what is the purpose of this resource ? are we creating the lambda in terraform or cloud formation ?

**Review by raanan245 — COMMENTED** [review]
Great work !

I've added my comments inline...  can you please have them resolved ?

File: main.tf (hunk 2 above)
Comment (evgenygigi) [reply main.tf:86]: datadog_integration_aws:
the integration provides the ability to collect logs of all the lambda functions on your aws account automatically. (without the need to subscribe to datadog forwarder)

But to collect logs from specific ECS log groups we need to subscribe to datadog forwarder.

File: README.md (hunk 1 above)
Comment (evgenygigi) [reply README.md]: Yes,
in next version we need to move the subscription to application level.

File: main.tf (hunk 2 above)
Comment (raanan245) [reply main.tf:86]: so we need to add a flag to the module to disable/enable the automatic collections of logs from all lambdas ?
In addition to specifying specific log-groups of the applications (ECS, or specific lambda/APIGW based services) ?

Comment (evgenygigi) [reply main.tf:86]: 1. As I've mentioned [here](https://github.com/toluna-terraform/terraform-aws-datadog-integration/blob/PLM-536-datadog-deployment-issue/README.md#datadog-log-collection-services) the flag is already exist and it covers the lambda functions which are not managed/created by terraform and their log groups are out of the context (the flag is optional).
2. This is the default scenario.

Comment (raanan245) [reply main.tf:86]: Thanks, please improve Readme  and documentation of each variable (as discussed verbally)

**Review by raanan245 — APPROVED** [review]

**Pull request merged**